### PR TITLE
feat: what's new in Kotlin 1.5.0

### PR DESCRIPTION
**Pull request by p7nov**

### CODE REVIEW

File: docs/topics/whatsnew15.md [hunk 1]
```diff
@@ -0,0 +1,740 @@
...
+The [kotlin-test](https://kotlinlang.org/api/latest/kotlin.test/) library introduces some new features:
+* [Simplified test dependencies usage in multiplatform projects](#simplified-test-dependencies-usage-in-multiplatform-projects)
+* [Automatic selection of a testing framework for Kotlin/JVM source sets](#automatic-selection-of-a-testing-framework-for-kotlin-jvm-source-sets)
+* [Assertion function updates](#assertion-function-updates)
+
+### Simplified test dependencies usage in multiplatform projects
+
+Now you can use the `kotlin-test` dependency to add dependencies for testing in the `commonTest` source set, and the Gradle plugin
+will infer the corresponding platform dependencies for each source set:
+* `kotlin-test-junit`/`kotlin-test-junit-5`/`kotlin-test-testng` for JVM source sets, see [automatic choice of a testing framework for Kotlin/JVM source sets](#automatic-selection-of-a-testing-framework-for-kotlin-jvm-source-sets)
```
Comment (PetrakovichVictoria) [inline docs/topics/whatsnew15.md:545]: @p7nov to our discussion in another PR:

>Please avoid using / as a conjunction. It's often unclear what you're replacing with it: and, or, something else?

Should I also fix it here?

File: docs/topics/whatsnew15.md [hunk 2]
```diff
@@ -0,0 +1,740 @@
...
+An existing kotlin-test setup with explicit dependencies will continue to work both in Gradle and in Maven.
+
+Learn more about [setting dependencies on test libraries](gradle.md#set-dependencies-on-test-libraries).
+
+### Automatic selection of a testing framework for Kotlin/JVM source sets
+
+The Gradle plugin now chooses and adds a dependency on a testing framework automatically. All you need to do is add
+the dependency `kotlin-test` in the common source set.
+
+Gradle uses JUnit 4 by default; therefore, the `kotlin('test')` dependency resolves to the variant for JUnit 4, namely `kotlin-test-junit`:
```
Comment (PetrakovichVictoria) [inline docs/topics/whatsnew15.md:561]: Should ```variant``` be removed here?

File: docs/topics/whatsnew15.md (hunk 1 above)
Comment (p7nov) [reply docs/topics/whatsnew15.md:545]: Yes. Let's leave only the default `-junit` in the beginning and explain other options before the link.

File: docs/topics/whatsnew15.md (hunk 2 above)
Comment (p7nov) [reply docs/topics/whatsnew15.md:561]: Yes, seems that it doesn't add value here.

File: docs/topics/whatsnew15.md (hunk 1 above)
Comment (PetrakovichVictoria) [reply docs/topics/whatsnew15.md:545]: >explain other options before the link

Can't catch what do you mean. Can discuss in DM.

**Pull request merged**